### PR TITLE
updates to firewall documentation based on today's run-through

### PR DESCRIPTION
**Pull request by acozine**
Reorganizes our notes into step-by-step instructions with examples. This could still use some refinement, but it's better than it was.

### CODE REVIEW

File: services/panos_fw.md [hunk 1]
```diff
@@ -86,7 +141,15 @@ Shared: Yes
 Description: Aliases
 IP Address: 128.112.###.###/32
 
-Services:
+## Advanced use cases
+
+These changes go beyond creating new VMs and opening them to standard traffic. If we add a new You will sometimes need ports not available from Palo Alto. Add and name the new port you want to open.
```
Comment (hackartisan) [inline services/panos_fw.md:146]: there's a partial sentence here

**Review by hackartisan — COMMENTED** [review]
Not a complete review but I did happen to notice a spot that needs editing

File: services/panos_fw.md (hunk 1 above)
Comment (acozine) [reply services/panos_fw.md]: Thanks for catching that @hackartisan!

**Review by VickieKarasic — APPROVED** [review]
This is a lot more readable now - thanks for organizing all this @acozine!

**Review by kayiwa — APPROVED** [review]
OK! Thanks for improving on them.

**Pull request merged**